src/components/ebay-pill/examples/02-selected/template.marko

```diff
@@ -1 +1 @@
-<ebay-pill checked=true>Pill</ebay-pill>
\ No newline at end of file
+<ebay-pill pressed=true>Pill</ebay-pill>
```

Comment:
Minor, but wondering why we are showing `pressed=true` instead of just `pressed`?